internal/v1/handler_blueprints.go

```diff
@@ -787,14 +894,37 @@ func (h *Handlers) FixupBlueprint(ctx echo.Context, id openapi_types.UUID) error
 		return err
 	}
 	desc := common.FromPtr(blueprintRequest.Description)
+	slog.DebugContext(ctx.Request().Context(), "starting buildServiceSnapshots during fixup",
+		"blueprint_id", blueprintEntry.Id,
+		"distribution", blueprintRequest.Distribution,
+		"has_openscap", blueprintRequest.Customizations.Openscap != nil)
```

Comment:
I think I do not see a nil check for `bpr.Customizations` above, this could be possibly a nil pointer error. If not, disregard.